### PR TITLE
Update assignments-in-teams.md

### PR DESCRIPTION
**Pull request by Jaqster**
Hi there, can someone approve this PR?  Made some edits to the MakeCode integration documentation.  Thanks!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/expand-teams-across-your-org/assignments-in-teams.md [hunk 1]
```diff
@@ -42,13 +42,15 @@ This setting is off by default.
 
 <a name="bkmakecode"> </a>
 ### MakeCode
-MakeCode is a block-based coding platform that brings computer science to life for all students. 
+Microsoft MakeCode is a block-based coding platform that brings computer science to life for all students. 
 
-This is a third party product or service that is subject to its own terms and privacy policy. You are responsible for your use of any third party products and services.
+MakeCode is a Microsoft product that is subject to the Microsoft [terms of use](https://go.microsoft.com/fwlink/?LinkID=206977) and [privacy](https://go.microsoft.com/fwlink/?LinkId=521839) policies.
```
Comment (JohanFreelancer9) [inline Teams/expand-teams-across-your-org/assignments-in-teams.md:47]:
```suggestion
MakeCode is a Microsoft product that is subject to the [Microsoft terms of use](https://go.microsoft.com/fwlink/?LinkID=206977) and the [Microsoft privacy statement](https://go.microsoft.com/fwlink/?LinkId=521839) policies.
```

File: Teams/expand-teams-across-your-org/assignments-in-teams.md [hunk 2]
```diff
@@ -42,13 +42,15 @@ This setting is off by default.
...
 
-This setting is off by default.
+This setting is off by default. To enable MakeCode assignments in Teams, in the **Teams Admin Center**, navigate to the **Assignments** section and turn the MakeCode toggle option to **On**.  Click **Save** and allow up to 24 hours for these settings to take effect.
```
Comment (JohanFreelancer9) [inline Teams/expand-teams-across-your-org/assignments-in-teams.md:49]:
```suggestion
This setting is off by default. To enable MakeCode assignments in Teams, in the **Teams Admin Center**, navigate to the **Assignments** section and turn the MakeCode toggle option to **On**. Click **Save** and allow up to 24 hours for these settings to take effect.
```

Comment (JohanFreelancer9) [reply Teams/expand-teams-across-your-org/assignments-in-teams.md:49]: Please remove extra space.

**Comment by Jaqster** [comment]
Looks good!  Thanks Johan.

**Comment by Jaqster** [comment]
Hi there, when can this be released?

**Pull request merged**